scalafix-core/src/main/scala-3/scalafix/util/Compat.scala

```diff
@@ -0,0 +1,9 @@
+package scalafix.util
+
+import scala.collection.immutable.IndexedSeq
+
+object Compat {
```

Comment:
unreferenced?

Reply:
I don't think so, also for the other `Compat` classes (at least in VS code) I don't get any references on the class level. Moreover if I remove the class I get a compilation error:
```
sbt:scalafix> core3/compile
[info] compiling 125 Scala sources to /Users/razvan.vacaru@TMNL.nl/srepo/scalafix/scalafix-core/target/jvm-3/classes ...
[error] -- [E008] Not Found Error: /Users/razvan.vacaru@TMNL.nl/srepo/scalafix/scalafix-core/src/main/scala/scalafix/util/TokenList.scala:7:21 
[error] 7 |import scalafix.util.Compat._
[error]   |       ^^^^^^^^^^^^^^^^^^^^
[error]   |value Compat is not a member of scalafix.util, but could be made available as an extension method.
[error]   |
[error]   |The following import might make progress towards fixing the problem:
[error]   |
[error]   |  import sourcecode.Text.generate
[error]   |
[error] one error found
[error] (core3 / Compile / compileIncremental) Compilation failed
```

Reply:
oh, sorry, I thought you had introduced a new Compat class, but this is just the scala-3 variant of an existing one :+1: